Makefile

```diff
@@ -65,18 +65,24 @@ ifeq ($(PULL_ARTIFACTS),no)
 endif
 
 build:
-ifeq ($(PULL_ARTIFACTS),yes)
-	python3 -u ./py/confluent/build/download_artifacts/download_artifacts.py
-endif
-
 ifeq ($(SKIP_TESTS),yes)
 	mvn -DskipTests=true install
 else
 	mvn install
 endif
 
-install: build
+newbuild:
+	git clone git@github.com:confluentinc/download-artifacts.git
+	python3 -u ./download-artifacts/py/confluent/build/download_artifacts/download_artifacts.py
+	rm -rf ./download-artifacts
+
+install:
+ifeq ($(PULL_ARTIFACTS),yes) 
+	make newbuild
```

Comment:
you can make the dependency conditional rather than invoking make indirectly, e.g. similar to like they use here https://stackoverflow.com/a/38353274/841827